plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -767,6 +767,11 @@ bool GSHacksDlg::OnMessage(UINT message, WPARAM wParam, LPARAM lParam)
 
 			EndDialog(m_hWnd, id);
 		} break;
+
+		case IDCANCEL:
+		{
+			EndDialog(m_hWnd, IDCANCEL);
+		} break;
```

Comment:
>You can't respond to comments in approved changes. :/
The scope braces aren't necessary, but it's like that in the rest of the file too. I copied the whole case from the shader dialog section earlier in the file.

See if commenting diff shows up in approved.

In most other cases in the file, the scope braces are only used when necessary. It's up to you. I only really haven't merged this because it's convenient for testing the review stuff.